acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/DrivingLicenceAPIPage.java

```diff
@@ -30,20 +40,25 @@ public class DrivingLicenceAPIPage {
     private static final Logger LOGGER =
             Logger.getLogger(DrivingLicenceAPIStepDefs.class.getName());
 
-    public void userIdentityAsJwtString(String criId)
+    public String getAuthorisationJwtFromStub(String criId)
             throws URISyntaxException, IOException, InterruptedException {
         String coreStubUrl = configurationService.getCoreStubUrl(false);
-
         if (coreStubUrl == null) {
             throw new IllegalArgumentException("Environment variable IPV_CORE_STUB_URL is not set");
         }
+        return getClaimsForUser(coreStubUrl, criId, LindaDuffExperianRowNumber);
```

Comment:
I think this should be taking row number from the feature file so that we can easily pre-populate with different user data

Reply:
Row number is now being passed as an argument in the test step.